### ISSUE TITLE
sphinx.ext.graphviz - Node URLs of inlined .svg images do not link to sphinx-doc sections

### ISSUE THREAD

**Issue by fkromer** [sphinx-doc/sphinx#2232]
I use sphinx.ext.graphviz to generate inline diagrams which are embedded in .rst input files with .. graphviz:: using the following conf.py section:

```
extensions = [
... ,
'sphinx.ext.graphviz',
]

...

# -- Options for Graphviz -------------------------------------------------
graphviz_dot = 'dot'
graphviz_dot_args = ['-Tsvg']
graphviz_output_format = 'svg'

...
```

In general the images are generated correctly as .svg images in the folder /_images/.svg (right clicking the image in the html output, options...).

My problem is that a diagram in the /design_languages.rst file

```
.. graphviz::

digraph build_automation_language{
  // styling
  size="11";
  node [color=lightblue2, style=filled];

  // 1st to 2nd level node transitions
  bal -> ph;
  bal -> cfg;
  bal -> nlp;
  bal -> log;
  bal -> dist;

  // 1st to 3rd level node transitions
  bal -> fh;

  // 2nd to 3rd level node transitions
  ph -> fh;
  nlp -> kwdp;
  nlp -> tp;
  nlp -> rkwp;
  log -> pblog;

  // 3rd to 4th level node transitions
  kwdp -> kwr;
  kwdp -> kwm;

  // nodes
  bal [label="Build Automation Lanugage"];
  ph [label="Path Handling", URL="../design_languages/build_automation_language.html#path-handling"];
  cfg [label="Configuration", URL="../design_languages/build_automation_language.html#configuration"];
  nlp [label="Natural Language Processing"];
  log [label="Logging"];
  dist [label="Distribution"];
  fh [label="File Handling", URL="../design_languages/build_automation_language.html#file-handling"];
  kwdp [label="Keyword Processing"];
  tp [label="Token Processing"];
  rkwp [label="Relative-to-Token Processing"];
  pblog [label="Pre-buffered Logging"];
  kwr [label="Keyword Replacement"];
  kwm [label="Keyword Modification"];
}

.. path-handling:

Path Handling
=============

.. configuration:

Configuration
=============

.. file-handling:

File Handling
=============
```

does create a .svg file in the related html output /_build/html/design_languages/build_automation_language.html but it does not hyperlink the sections .. path-handling:, .. configuration: and .. file-handling:.

For reference:
- latest .html output: http://eswp3.readthedocs.org/en/latest/design_languages/build_automation_language.html
- related source code: https://github.com/fkromer/eswp3/blob/master/design_languages/build_automation_language.rst.

**Comment by fkromer** [comment]
For reference:
- latest .html output: http://eswp3.readthedocs.org/en/latest/design_languages/build_automation_language.html
- related source code: https://github.com/fkromer/eswp3/blob/master/design_languages/build_automation_language.rst.

**Comment by tk0miya** [comment]
Since version 1.3.4 (just released yesterday), Sphinx supports hyperlinks in graphviz's SVG output.
Please wait for updating Sphinx on readthedocs.org and try it again.

FYI:
sphinx.ext.graphviz requires to use `target` attribute to create link.
More details, please see http://www.sphinx-doc.org/en/stable/ext/graphviz.html#confval-graphviz_output_format .

Thanks,

**Comment by tk0miya** [comment]
MEMO: It is fixed at #2176

**Comment by fkromer** [comment]
Great! Thanks a lot!
